include/cute/arch/util.hpp

```diff
@@ -36,39 +36,35 @@
 
 #if defined(__clang__) && defined(__CUDA__)
   //  __cvta_generic_to_shared was added in Clang 14: https://reviews.llvm.org/D111665
-  #define CUTE_CLANG_SUPPORTS_CVTA_GENERIC_TO_SHARED (__clang_major__ >= 14)
+#  define CUTE_CLANG_SUPPORTS_CVTA_GENERIC_TO_SHARED (__clang_major__ >= 14)
 
-  #ifndef _WIN32
+#  ifndef _WIN32
   // __nvvm_get_smem_pointer added in Clang 14: https://reviews.llvm.org/D111665
-    #define CUTE_CLANG_SUPPORTS_NVVM_GET_SMEM_POINTER (__clang_major__ >= 14)
-  #else
+#    define CUTE_CLANG_SUPPORTS_NVVM_GET_SMEM_POINTER (__clang_major__ >= 14)
+#  else
     // ... but will not work on Windows until Clang 15: https://reviews.llvm.org/D122897
-    #define CUTE_CLANG_SUPPORTS_NVVM_GET_SMEM_POINTER (__clang_major__ >= 15)
-  #endif
+#    define CUTE_CLANG_SUPPORTS_NVVM_GET_SMEM_POINTER (__clang_major__ >= 15)
+#  endif
 #endif
 
 #if defined(__NVCC__) || defined(__CUDACC_RTC__)
   // __cvta_generic_to_shared added in CUDA 11+
-  #if defined(__CUDA_ARCH__) && (__CUDACC_VER_MAJOR__ >= 11)
-    #define CUTE_NVCC_SUPPORTS_CVTA_GENERIC_TO_SHARED 1
-  #endif
+#  define CUTE_NVCC_SUPPORTS_CVTA_GENERIC_TO_SHARED (__CUDACC_VER_MAJOR__ >= 11)
```

Comment:
we need to write
```
#if xxx
  #define xxx 1
#endif
```
otherwise, some compiler will yell.

Reply:
Done -- changed all checks to `#define VAR 1` only if the associated condition is true.